docs/running-rules.md

```diff
@@ -9,39 +9,44 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. Use Semgrep rules in the following ways:
 
-Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
+- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them.
+- Local rules:
+  - Create and run [ephemeral rules](#ephemeral-rules) in the command line.
+  - Create and run [YAML-defined rules](#creating-and-using-yaml-defined-rules-file).
+- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
+
+You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI.
 
+The following diagram displays the current number of Semgrep rules:
 <div className="lang-container" style={{marginBottom: '20px'}}>
   <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
 </div>
 
-You can run existing and custom Semgrep rules locally with the Semrgep command line tool or continuously with Semgrep CI. See [Getting started](../getting-started/) for their respective installation and setup.
-
-## Run Registry rules
-
-Explore the [Semgrep Registry](https://semgrep.dev/explore) and run rules and rulesets via:
+## Running Semgrep Registry rules
```

Comment:
I didn't see anywhere talking about running rules non-locally. If we don't, depending on how things get structured, you might want to change this title to "Running Semgrep Registry rules locally", and then you can take out that first sentence below. Also if we want to talk about running not locally, can we link to the CI page if people want that?

Reply:
Ok, changes done as you suggested, makes sense! I have one uncertainty though. To run Semgrep non-locally, we need to integrate Semgrep App with CI, right? Not sure if CI overview is so much helpful here. Maybe Semgrep App getting started might be a nicer link here? :-) @efortuna

Reply:
hmm I still think the CI overview page https://semgrep.dev/docs/semgrep-ci/overview says you need the app and it makes the most sense to link to the CI page. But I defer to @s-santillan for the final opinion.

Reply:
@efortuna adding this as a subsection, so under the step 4. in the procedure described in this section, there will be:

```markdown
### Next steps

To use Semgrep Registry continuously in your CI/CD pipeline, see the documentation for [Semgrep in CI](https://semgrep.dev/docs/semgrep-ci/overview).
```